recipes/centos_vertx/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM registry.centos.org/centos/centos
+
+MAINTAINER Clement Escoffier
+
+ARG JAVA_VERSION=1.8.0
+ARG VERTX_VERSION=3.3.3
+
+RUN yum -y update && \
+    yum -y install sudo openssh-server centos-release-scl && \
+    yum clean all
+
+RUN yum -y install rh-maven33 java-${JAVA_VERSION}-openjdk-devel && \
```

Comment:
can we group all the RUN instructions ? (squashing layers during build is still experimental)